students/WesleyM/Lesson3/StringFormatLab.py

```diff
@@ -0,0 +1,52 @@
+def task_one():
+    """Converts (2, 123.4567, 10000, 12345.67) to 'file_002 :   123.46, 1.00e+04, 1.23e+04'"""
+    string = (2, 123.4567, 10000, 12345.67)
```

Comment:
I think it's a little confusing to name the tuple `string`, and in this case you could just pass the tuple straight to `format()` as args, since you're not changing the order of the items:

`print("file_{:03d}: {:.2f}, {:.2e}, {:.2e}".format(2, 123.4567, 10000, 12345.67))`